wp_api/src/login/login_client.rs

```diff
@@ -239,8 +258,7 @@ impl WpLoginClient {
         }
     }
 
-    // Fetches the site's homepage with a HEAD request, then extracts the Link header pointing
-    // to the WP.org API root
+    // Fetches the site's homepage with a HEAD request
```

Comment:
```suggestion
    // Fetches the site's homepage headers with a HEAD request
```

Reply:
Addressed in 30cfa0427745e558e20a4d9cca693aa45e468e6c.